src/Sylius/Bundle/AddressingBundle/phpunit.xml.dist

```diff
@@ -13,6 +13,7 @@
     <testsuites>
         <testsuite name="SyliusAddressingBundle Test Suite">
             <directory>./test/</directory>
+            <directory>./Tests/</directory>
```

Comment:
We should put these tests in a one folder I suppose

Reply:
Not really, we use the following convention in other bundles (as Symfony does) and in fact `test` should be somehow merged to `Tests` (and merging it back to `test` breaks autoloading).

Reply:
It is one of the solutions. I don't say, that we have to merge the new one with the existing one, but on the other hand we have a bundle test there. Just a thing, that we should discuss before we introduce too much test conventions in our test suite.